sonic-xcvrd/scripts/xcvrd

```diff
@@ -183,21 +269,42 @@ def post_port_dom_info_to_db(logical_port_name, table, stop=threading.Event()):
-                                                  ('tx2bias', dom_info_dict['tx2bias']),
-                                                  ('tx3bias', dom_info_dict['tx3bias']),
-                                                  ('tx4bias', dom_info_dict['tx4bias']),
-                                                  ('tx1power', dom_info_dict['tx1power']),
-                                                  ('tx2power', dom_info_dict['tx2power']),
-                                                  ('tx3power', dom_info_dict['tx3power']),
-                                                  ('tx4power', dom_info_dict['tx4power'])])
+                beautify_dom_threshold_info_dict(dom_info_dict)
+                fvs = swsscommon.FieldValuePairs(
+                        [('temperature', dom_info_dict['temperature']),
+                         ('voltage', dom_info_dict['voltage']),
+                         ('rx1power', dom_info_dict['rx1power']),
+                         ('rx2power', dom_info_dict['rx2power']),
+                         ('rx3power', dom_info_dict['rx3power']),
+                         ('rx4power', dom_info_dict['rx4power']),
+                         ('tx1bias', dom_info_dict['tx1bias']),
+                         ('tx2bias', dom_info_dict['tx2bias']),
+                         ('tx3bias', dom_info_dict['tx3bias']),
+                         ('tx4bias', dom_info_dict['tx4bias']),
+                         ('tx1power', dom_info_dict['tx1power']),
+                         ('tx2power', dom_info_dict['tx2power']),
+                         ('tx3power', dom_info_dict['tx3power']),
+                         ('tx4power', dom_info_dict['tx4power']),
+                         ('temphighalarm', dom_info_dict['temphighalarm']),
+                         ('temphighwarning', dom_info_dict['temphighwarning']),
+                         ('templowalarm', dom_info_dict['templowalarm']),
+                         ('templowwarning', dom_info_dict['templowwarning']),
+                         ('vcchighalarm', dom_info_dict['vcchighalarm']),
+                         ('vcchighwarning', dom_info_dict['vcchighwarning']),
+                         ('vcclowalarm', dom_info_dict['vcclowalarm']),
+                         ('vcclowwarning', dom_info_dict['vcclowwarning']),
+                         ('rxpowerhighalarm', dom_info_dict['rxpowerhighalarm']),
+                         ('rxpowerlowalarm', dom_info_dict['rxpowerlowalarm']),
+                         ('rxpowerhighwarning', dom_info_dict['rxpowerhighwarning']),
+                         ('rxpowerlowwarning', dom_info_dict['rxpowerlowwarning']),
+                         ('txbiashighalarm', dom_info_dict['txbiashighalarm']),
+                         ('txbiaslowalarm', dom_info_dict['txbiaslowalarm']),
+                         ('txbiashighwarning', dom_info_dict['txbiashighwarning']),
+                         ('txbiaslowwarning', dom_info_dict['txbiaslowwarning'])
+                        ])
```

Comment:
why we need the threshold here since you already have post_port_dom_threshold_info_to_db() dedicated for them?

Reply:
Hi @keboliu 

Removed the irrelevant code from this.